### DISCUSSION TITLE
2023년 05월 22일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#57] [General]
# 2023년 05월 22일

[ 2023-05-01 ] [ 2023-05-02 ] [ 2023-05-04 ] [ 2023-05-07 ] [ 2023-05-08 ] [ 2023-05-10 ] [ 2023-05-11 ] [ 2023-05-12 ] [ 2023-05-14 ] [ 20…

http://vero.wiki/2023-05-22/

**Comment by kevstevie** [comment]
오늘 알차네요

**Comment by drunkenhw** [comment]
통합테스트 덕에 커버리지가 높은게 아닐까요?

**Reply by Cyma-s** [reply]
들켰네;;
